src/ILCompiler.Compiler/src/Compiler/DelegateCreationInfo.Sorting.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Internal.TypeSystem;
+
+namespace ILCompiler
+{
+    /// <summary>
+    /// Captures information required to generate a ReadyToRun helper to create a delegate type instance
```

Comment:
XmlDoc comments should not be repeated. VS picks one at random and comments go stale all the time even without the duplication's help